pandas/tests/resample/test_datetime_index.py

```diff
@@ -694,6 +694,20 @@ def test_resample_timestamp_to_period(simple_date_range_series):
     tm.assert_series_equal(result, expected)
 
 
+def test_datetime_resample_sum_min_count():
```

Comment:
can you move this to the base class where its tests all of DTI, PI, TDI; pls use the fixtures.